apps/measures/models.py

```diff
@@ -125,3 +149,191 @@ class MeasuresOverviewPage(MetadataPageMixin, Page):
         index.SearchField('page_intro_en', partial_match=True),
         index.SearchField('page_intro_de_ls', partial_match=True),
     ]
+
+
+class MeasuresDetailPage(Page):
+    page_title_de = models.CharField(
+        max_length=120, blank=False)
+    page_title_en = models.CharField(
+        max_length=120, blank=True)
+    page_title_de_ls = models.CharField(
+        max_length=120, blank=True)
+
+    responsible_orgs_de = models.CharField(
+        max_length=80, blank=False)
+    responsible_orgs_en = models.CharField(
+        max_length=80, blank=True)
+    responsible_orgs_de_ls = models.CharField(
+        max_length=80, blank=True)
+
+    involved_orgs_de = models.CharField(
+        max_length=300, blank=True)
+    involved_orgs_en = models.CharField(
+        max_length=300, blank=True)
+    involved_orgs_de_ls = models.CharField(
+        max_length=300, blank=True)
+
+    districts = MultiSelectField(
+        blank=True,
+        max_choices=3,
+        choices=BEZIRK_CHOICES
+    )
```

Comment:
@khamui sorry just experienced this issue, maybe help text for may choice is good idea?

Reply:
yes, added. not sure about my wordin, though.